lib/files.cf

```diff
@@ -428,7 +428,7 @@ bundle edit_line manage_variable_values_ini(tab, sectionName)
       ifvarclass => "edit_$(cindex[$(index)])";
 
       # match a line starting like the key something
-      "$(index)\s*=.*"
```

Comment:
Please add 

```
Ticket: CFE-3244
```

to the commit message, and it will be included in the changelog.

Reply:
@olehermanse So should I squash it as well? Into one commit?

Reply:
@oleorhagen No, why do you say that?

Reply:
Because of what Craig said above...

Reply:
I don't know exactly what he means there. But you should add a ticket number to the commit that adds a changelog entry :)

Reply:
@oleorhagen and make the Changelog entry past tense(!) 😠 

```
Whitespace is now allowed at the beginning of ini key-values
```

Reply:
haha, allowed is past tense, no? :p 
Anywho. Do you have any answers to the questions I posted for Nick, @olehermanse ?

Reply:
Like, I don't think this is ready to merge as is.